app/src/ui/hooks/useTheme.ts

```diff
@@ -11,11 +11,11 @@ const REG_EXP = new RegExp(`${THEME_PREFIX}\\w+`);
 export function useTheme() {
   const themeCache = new LocalStorage(THEME_NAMESPACE_KEY);
 
-  function getClassNameFromTheme(theme) {
+  function getClassNameFromTheme(theme: string) {
```

Comment:
Create an enum containing names of themes: 'light' and 'dark.'

Then reuse this enum in every place where we handle the theme

So this means that in this specific case, you gonna apply not a string but an enum type